adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,109 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5_2']
+=== Release 1.5.2, February 2020
+
+[id='sec.1_5_2.new']
+==== What Is New?
+* {scf} has been updated to version 2.20.3:
+** cf-deployment has been updated to 12.17
+** Optimized startup time for various roles
+** Added/fixed podAntiAffinity rules for various roles
+* The Stratos UI has been updated to version 2.7.0:
+** Add support for {caasp} V4.1 and {k8s} 1.16+ environments
+** Show last operation and service broker info for service instances
+** For a full list of features and fixes see https://github.com/SUSE/stratos/blob/v2-master/CHANGELOG.md#270.
+* The Stratos Metrics has been updated to version 1.1.2:
+** Add support for {caasp} V4.1 and {k8s} 1.16+ environments
+** For a full list of features and fixes see https://github.com/SUSE/stratos-metrics/blob/master/CHANGELOG.md#112.
+
+[id='sec.1_5_2.feature']
+==== Features and Fixes
+* Set `topologyKey` for `podAntiAffinity` rule to `kubernetes.io/hostname`
+* Inform operator of the need for CSR approval for Eirini internal registry in Helm notes
+* Pod restarts for many roles during UAA and SCF startup lessened for faster deployment
+* Expanded GoRouter ciphersuite to be compatible with a wider range of clients
+* Made Helm chart compatible with {k8s} 1.16
+* Use system cert store if `UAA_CA_CERT` is not specified
+* Ensure the `HTTP_PORT` is exposed for `file_server` to ensure upgrades do not require longer app downtime
```

Comment:
This is an internal detail that doesn't need to be listed. It is also not a regression itself, just makes upgrades possible with running apps.